File: CarlosTriana_GenerarTiempos.cpp
#include <ctime>
#include <iostream>
using namespace std;

// se define una funcion de fibonacci que retorna el n-esimo valor 
int fibonacci(int N){
	if(N==1||N==0){
	return(N);
	}
	else{
	return(fibonacci(N-1)+fibonacci(N-2));
	}
}
//se define una funcion que calcula el tiempo de calculo para el n-esimo valor de la funcion de fibonacci
float get_time(int N){
	clock_t temp;
	temp = clock();

	fibonacci(N);
	temp = clock()-temp;

	float tiempo = ((float)temp)/CLOCKS_PER_SEC;

return tiempo; 
}

//imprime el tiempo de calculo
int main(){
	int N=35;
	int n=0;
		while(n<N)
{
		cout<<n<<","<<get_time(N)<<"\n";
			n++;  
		}
return 0;		
};

File: CarlosTriana_GenerarTiempos.py
import numpy as np
import time

#se define la funcion de fobonacci que retorna el N-esimo numero de esta secuencia
def fibonacci(N):
	if (N==1 or N==0):
		return N
	else:
		return fibonacci(N-1)+fibonacci(N-2)

#funcion que calcula el tiempo que tarda en calcular el n-esimo valor de la secuencia de fibonacci
		
t0 = time.time()
def get_time(N):
	
	fibonacci(N)
	
	t1 = time.time()-t0
	
	return t1

print "el tiempo que tarda en obtener el valor",N, "es",get_time(35)
